src/integration/pages/onDemandRadioEpisode/ampTests.js

```diff
@@ -1,6 +1,8 @@
 import { runCoreAmpTests, runAmpAnalyticsTests } from '../../common';
+import runAmpOnlyEpisodeTests from './ampOnlyEpisodeTests';
 
 export default () => {
   runCoreAmpTests();
   runAmpAnalyticsTests();
+  runAmpOnlyEpisodeTests();
```

Comment:
can we just put the content of `ampOnlyEpisodeTests.js` here?

Reply:
👍 - done the same for `canonicalTests.js`